src/main/java/org/jitsi/jigasi/transcription/Transcriber.java

```diff
@@ -170,45 +175,99 @@ public Transcriber(TranscriptionService service)
     /**
      * Add a participant to the list of participants being transcribed
      *
-     * @param chatMember the chat participant to be added
-     * @param ssrc the ssrc of the participant to be added
+     * @param identifier the identifier of the participant
      */
-    public void add(ChatRoomMember chatMember, long ssrc)
+    public void participantJoined(String identifier)
     {
-        Participant participant;
-        if (this.participants.containsKey(ssrc))
+        if (this.participants.containsKey(identifier))
```

Comment:
This code will throw a NPE if the participant is removed between the `containsKey` check and the call to `get`. I would just call `get` once and then check for `null`.